src/libunwind/build.rs

```diff
@@ -5,14 +5,14 @@ fn main() {
     let target = env::var("TARGET").expect("TARGET was not set");
 
     if cfg!(feature = "llvm-libunwind") &&
-        (target.contains("linux") ||
+        (target.contains("linux") && !target.contains("musl") ||
```

Comment:
might want to add some parens wrt precendence between this `&&` and the `||` next to it?

Reply:
Done.

Reply:
Done